lxd/instance/drivers/driver_qemu.go

```diff
@@ -1207,6 +1207,16 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 		}
 	}()
 
+	// Define a set of files to open that must wait until QEMU is called to be closed.
```

Comment:
In general I would prefer to see the commits in this PR further broken down into smaller logical commits.

For example, this commit ` lxd/instance/drivers: Close files opened during socket path handling ` is altering the behaviour of both the config drive and TPM devices - it would be preferable to see these as separate commits so its easier to reason about.

Also the PR title `Use shorter virtiofsd socket paths` seems now to be out of date compared with the current implementation. 

Thanks

Reply:
`lxd/instance/drivers: Close files opened during socket path handling` was dropped but I broke down `lxd/instance: Use shortenedSocketPath when passing socket paths to qemu` into two commits, one for virtiofsd sockets and one for the swtpm socket